src/main/java/core/basesyntax/service/impl/CsvDataWriterImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.DataWriter;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class CsvDataWriterImpl implements DataWriter {
+    private static final String PATH = "src/main/resources";
+    private final DataConverterImpl dataConverter;
+
+    public CsvDataWriterImpl() {
+        this.dataConverter = new DataConverterImpl();
+    }
+
+    @Override
+    public void writeToFile(List<String> report, String filename) {
+        File file = new File(PATH + File.separator + filename);
+        String lines = dataConverter.convertToStringFormat(report);
```

Comment:
```suggestion
    @Override
    public void writeToFile(String report, String filename) {
        File file = new File(PATH + File.separator + filename);
```
Let’s avoid dependency. Pass report to this method